### ISSUE TITLE
Make sure browser works with screen readers

### ISSUE THREAD

**Issue by aslagle** [NYPL-Simplified/opds-browser#33]

**Comment by skomputer** [comment]
Partial accessibility checklist, based on WebAIM's [Introduction to Web Accessibility](http://webaim.org/intro/):

- alt attributes for book covers
- use heading tags for collection titles, lane titles, book titles, links
- use lists for sets of lanes and books
- labels for form fields?

**Comment by jce1028** [comment]
if you need would ever have questions around this, there are two technical resources who may be helpful in regards to accessibility needs.

Becky Gibson of IBM  - gibson.becky@gmail.com   - she is the lead developer at IBM for accessibility in their interfaces.  She works with Readium on accessibility.

Daniel Weck - daniel.weck@gmail.com  - leads the accessibility issues for Readium and the Daisy Consortium.  

You can reach them on Readium's slack project on the a11y channel.  You can get on their project via the following link.  https://readium-slack.herokuapp.com/  and then join the A11y channel.  Daniel is also a member of our slack project and channel.
